src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/CopyNumberTriStateTransitionProbabilityCache.java

```diff
@@ -137,8 +137,16 @@ public double visit(final int to, final int from, final double probability) {
 
     /**
      * Wrapper for a {@link RealMatrix} that is indexed by the CopyNumberTriState enum
+     *
```

Comment:
Not sure what this is all about.  Is this an issue with `RealMatrix`?  Maybe @lbergelson or @LeeTL1220 can comment.